src/rgw/driver/sfs/object.cc

```diff
@@ -282,26 +287,98 @@ void SFSObject::gen_rand_obj_instance_name() {
   state.obj.key.set_instance(buf);
 }
 
-int SFSObject::get_obj_attrs(
-  optional_yield y,
-  const DoutPrefixProvider *dpp,
-  rgw_obj *target_obj
-) {
-  lsfs_dout(dpp, 10) << " TODO" << dendl;
+/*
+  This should be intended as a mere fetch of object's attributes.
+  The caller will likely call get_attrs() after get_obj_attrs().
+  This is how we have interpreted this call.
+  The average usage of this seems to suggest that one first calls
```

Comment:
what is average usage?

Reply:
Not sure to have understood correctly what are you asking here, btw: for "average usage" is what I have interpreted as "a frequent usage case" for `get_obj_attrs` method.

Reply:
The sentence is worded very vague and passive. We can know how calls what. We should know. Especially if we derive calling semantics from that. So, what is the average usage? There is possibly just one usage in the RGWOps

Reply:
the problem is that the sentence `is worded very vague and passive` because it is! :grin:
It represents perfectly our uncertainty.

Reply:
PS:feel  free to investigate deeper the topic if you think it is worth to.